pkg/project/project.go

```diff
@@ -34,6 +34,14 @@ func Create(client *occlient.Client, projectName string) error {
 	return nil
 }
 
+func Delete(client *occlient.Client, projectName string) error {
```

Comment:
It will be good to add unit test for this

Reply:
+1

Reply:
> It will be good to add unit test for this

Ack Done

> unit test for this, https://github.com/redhat-developer/odo/blob/9cb001ec7d86d39d06084c0247a9cf762824323e/pkg/occlient/occlient.go#L1151

Func remains Unaltered by me :)